authentication/account/worker/user_deactivation_notification_worker.go

```diff
@@ -0,0 +1,116 @@
+package worker
+
+import (
+	"context"
+	"time"
+
+	"cirello.io/pglock"
+
+	"github.com/fabric8-services/fabric8-auth/application"
+	"github.com/fabric8-services/fabric8-auth/log"
+	worker "github.com/fabric8-services/fabric8-auth/worker/repository"
+)
+
+// UserDeactivationNotificationWorker the interface for the User Deactivation Worker,
+// which takes care of deactivating accounts of inactive users who were previously notified but did not come back afterwards.
+type UserDeactivationNotificationWorker interface {
+	Start(freq time.Duration)
+	Stop()
+}
+
+// NewUserDeactivationNotificationWorker returns a new UserDeactivationNotificationWorker
+func NewUserDeactivationNotificationWorker(ctx context.Context, app application.Application) UserDeactivationNotificationWorker {
+	return &userDeactivationNotificationWorker{
+		ctx:   ctx,
+		app:   app,
+		owner: worker.GetLockOwner(ctx),
+	}
+}
+
+type userDeactivationNotificationWorker struct {
+	ctx    context.Context
+	app    application.Application
+	lock   *pglock.Lock
+	ticker *time.Ticker
+	stopCh chan bool
+	owner  string
+}
+
+// Start starts the worker with the given timer
+func (w *userDeactivationNotificationWorker) Start(freq time.Duration) {
```

Comment:
Where is the worker started from, and what is the default value for `freq` ?

Reply:
the `Start(time.Duration)` func must be called in `main.go` (not implemented yet, actually)